js/index.js

```diff
@@ -8,35 +8,140 @@ const siteContent = {
+logo.setAttribute("src", siteContent["nav"]["img-src"]);
+
+//NAV
+
+let headerNav = document.querySelectorAll("nav a");
+headerNav.forEach((item, num) => {
+  item.innerText = siteContent.nav[`nav-item-${num + 1}`];
+  item.setAttribute("style", "color: green");
+});
+
+let anchorContainer = document.querySelector("nav");
+
+let firstAnchor = document.createElement("a");
+
+firstAnchor.setAttribute("href", "#");
+firstAnchor.setAttribute("style", "color: green");
+firstAnchor.innerText = "Hyrule!";
+
+anchorContainer.prepend(firstAnchor);
+
+let lastAnchor = document.createElement("a");
+
+lastAnchor.setAttribute("href", "#");
+lastAnchor.setAttribute("style", "color: green");
+lastAnchor.innerText = `I <3 Zelda!`;
+
+anchorContainer.appendChild(lastAnchor);
+
+
+
+//CTA
+
+let ctaHeader = document.querySelector(".cta h1");
+ctaHeader.innerText = siteContent.cta.h1;
+
+let ctaButton = document.querySelector(".cta button");
+ctaButton.innerText = siteContent.cta.button;
+
+let ctaImg = document.querySelector(`#cta-img`);
+ctaImg.setAttribute(`src`, siteContent.cta["img-src"]);
+
+// MAIN CONTENT - TOP CONTENT - H4
+
+let topContentHeader = document.querySelectorAll(".top-content h4");
+topContentHeader[0].innerText = siteContent["main-content"]["features-h4"];
+
+topContentHeader[1].innerText = siteContent["main-content"]["about-h4"];
+
+// MAIN CONTENT - TOP CONTENT - P
+
+let topContentParagraph = document.querySelectorAll(".top-content p");
+topContentParagraph[0].innerText =
+  siteContent["main-content"]["features-content"];
+
+topContentParagraph[1].innerText = siteContent["main-content"]["about-content"];
+
+// MAIN CONTENT - MIDDLE IMAGE
+
+let middleImage = document.querySelector(`#middle-img`);
+middleImage.setAttribute(`src`, siteContent["main-content"]["middle-img-src"]);
+
+// MAIN CONTENT - BOTTOM CONTENT - H4
+
+let bottomContentHeader = document.querySelectorAll(".bottom-content h4");
+
+bottomContentHeader[0].innerText = siteContent["main-content"]["services-h4"];
+
+bottomContentHeader[1].innerText = siteContent["main-content"]["product-h4"];
+
+bottomContentHeader[2].innerText = siteContent["main-content"]["vision-h4"];
+
+// MAIN CONTENT - BOTTOM CONTENT - P
+
+let bottomContentParagraph = document.querySelectorAll(".bottom-content p");
+
+bottomContentParagraph[0].innerText =
+  siteContent["main-content"]["services-content"];
+
+bottomContentParagraph[1].innerText =
+  siteContent["main-content"]["product-content"];
+
+bottomContentParagraph[2].innerText =
+  siteContent["main-content"]["vision-content"];
+
+// CONTACT H4
+
+let contactHeader = document.querySelector(".contact h4");
+contactHeader.innerText = siteContent.contact["contact-h4"];
+
+// CONTACT P
+
+let contactParagraphs = document.querySelectorAll(`.contact p`);
+
+contactParagraphs[0].innerText = siteContent.contact.address;
+contactParagraphs[1].innerText = siteContent.contact.phone;
+contactParagraphs[2].innerText = siteContent.contact.email;
```

Comment:
From 90 to here, you can spot some similar logic being repeated.  Can you think of a way you might be able to write this as a loop?